resource_calendar_schedule_iteration/hooks.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+import pytz
+import datetime
+
+from datetime import timedelta
+
+from odoo.addons.resource.models.resource import ResourceCalendar
+from odoo.tools.float_utils import float_compare
+
+
+def to_naive_user_tz(datetime, record):
```

Comment:
@jbeficent Why do you redefine this method already defined into `odoo.addons.resource.models.resource` ?

Why not
`from odoo.addons.resource.models.resource import to_naive_user_tz`

Reply:
Agree